includes/wikia/upload/WikiaUploadStashFile.class.php

```diff
@@ -39,4 +39,14 @@ public function getThumbUrl( $suffix = false ) {
 		}
 		return $path;
 	}
+
+	/**
+	 * Get an URL to access the original stashed file
+	 *
+	 * @return string
+	 */
+	public function getOriginalFileUrl() {
+		return $this->repo->getZoneUrl( 'temp' ) . '/' . $this->getUrlRel();
```

Comment:
@nandy-andy do you know why the `temp` is being added here? It's adding an arbitrary path to the directory structure. It's causing https://wikia-inc.atlassian.net/browse/PLATFORM-824.